Library/ViewModels/PledgeViewModel.swift

```diff
@@ -482,8 +482,6 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
       .switchMap { [checkoutIdProperty] input in
         AppEnvironment.current.apiService.createBacking(input: input)
           .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
-          // FIXME: Remove once UpdateBacking mutation is aligned on error type.
```

Comment:
As promised, this could now be removed as both mutations share the same error type. This also means that against the main branch there are no changes to `PledgeViewModel`.